.buildkite/longruns/pipeline.yml

```diff
@@ -119,32 +119,32 @@ steps:
     steps:
 
       - label: ":computer: held suarez (ρe_tot) third order"
-        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --upwinding third_order --dt 350secs --t_end 1200days --fps 30 --job_id longrun_hs_rhoe_third --dt_save_to_sol 1days --dt_save_to_disk 10days"
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --tracer_upwinding third_order --dt 350secs --t_end 1200days --fps 30 --job_id longrun_hs_rhoe_third --dt_save_to_sol 1days --dt_save_to_disk 10days"
```

Comment:
It's good to test some of these with `third_order` just to see if it improved relative to when we were upwinding both energy and tracers with the same upwinding mode, but ultimately, we want to use FCT since it is supposed to reduce oscillations. This PR should definitely add some `--tracer_upwinding zalesak` mode jobs

Reply:
I just realized that this test is meaningless because dry Held-Suarez has no tracers. I will replace it with a moist Held-Suarez test that uses Zalesak FCT.